controller/config/config.go

```diff
@@ -25,13 +25,20 @@ import (
 	"github.com/urfave/cli"
 )
 
+// Database types
+const (
+	DatabaseTypeRedis  = "redis"
+	DatabaseTypeMemory = "memory"
+	DatabaseTypeK8S    = "k8s"
```

Comment:
If this is exposed to the user, I think we're using  `kubernetes` elsewhere in config/flag values, so consider changing for consistency. Should be reflected in docs too.